leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/impl/ObservationServiceImpl.java

```diff
@@ -230,6 +232,8 @@ public void onNotification(Request coapRequest, Response coapResponse) {
             return;
         }
 
+        clientStatusTracker.setClientOnline(registration);
```

Comment:
I am not fully convinced of notifying the Queue mode status listeners here because this line will not be invoked if there are no `ObservationListener` registered. But adding this in the very begining means that the registrationStore has to invoked at the very begining.